narwhals/series.py

```diff
@@ -464,6 +464,33 @@ def ewm_mean(
                2.428571
             ]
 
+            pandas and Polars handle nulls differently. So, calculating ewm over
+            a sequence with null values leads to distinct results:
```

Comment:
I think it's that Polars preserves null values, whereas pandas forward-fills

Can we preserve null values for pandas too?

Reply:
I don't think you need to fill the nulls, but just preserve them - so, if a value was null to start with, it should be null in the result too

we do something like that in `timestamp`

Reply:
Doing like you do in `timestamp` solves it for `None`.. but with a series like `[1.0, float("nan"), 4.0]` we still have this:

Pandas:
```
0    1.0
1    NaN
2    3.4
dtype: float64
```

Polars:
```
shape: (3,)
Series: '' [f64]
[
        1.0
        NaN
        NaN
]
```

Reply:
sure, cause Polars treat 'nan' differently from null - but if we use the null value for both, does the result match? e.g. `[1., None, 4.]`?

Reply:
Yes, with a Series like `[1.0, None, float("nan"), 4.0]` we get this:

```
pandas:

0    1.000000
1         NaN
2         NaN
3    3.666667

polars:

dtype: float64
shape: (4,)
Series: '' [f64]
[
        1.0
        null
        NaN
        NaN
]
```

Reply:
yeah I think just using `None` in the data that you pass to the constructor in the test should be fine

Reply:
Test modified.. 
To resume then:  If someone uses a pandas df, they won't get pandas normal results. We match polars behaviour when "None" is in the input, but we don't when there is "NaN". 🤯 .. 😁

CI failing with older versions but I can't reproduce locally.

Reply:
for Polars 'nan' is only the result of illegal mathematical operations (like 0/0) so it's far rarer to encounter it there

regarding older versions ci - i'd suggest making a separate virtual environment and installing the versions which show up in the `show deps` step of the ci job

Reply:
Older versions of polars give similar results to pandas when there is a null (`None`).

So with an input of: `{"a": [2.0, 4.0, None, 3.0]}` Then:
```
Expected: {'a': [2.0, 3.3333333333333335, nan, 3.142857142857143]}
Got: {'a': [2.0, 3.3333333333333335, 3.3333333333333335, 3.142857142857143]}
```

For the moment I'm "xfailing" that test with older versions.  I'm not sure if that's correct.

Reply:
thanks - is it possible to use `pl.when` to preserve the null values for old versions of Polars?

either that, or raise `NotImplementedError` for old versions of Polars for now, and let's create an issue to track preserving null values in old Polars versions

Reply:
Thanks for the review.. I added the "raise" for now. I'll do the follow up PR.

Reply:
thanks!